### DISCUSSION TITLE
Peak value

### DISCUSSION THREAD

**Discussion by LukasVFL99** [tbnobody/OpenDTU#699] [Ideas]
Is it possible to display the Peak kWh from each string? So the highest value at some time of the day? And what about a Diagram for reading the values?

**Comment by DPO99** [comment]
Hi, it is possible via [MQTT](https://github.com/tbnobody/OpenDTU/blob/master/docs/MQTT_Topics.md) or [API](https://github.com/tbnobody/OpenDTU/blob/master/docs/Web-API.md). You can see each String incl. the Details. You only need a System to log this informations (IoBroker, HomeAssistent or any other HomeServer).

BR
Timo
